util-images/access-tokens/README.md

```diff
@@ -0,0 +1,26 @@
+# Access-tokens
+
+Utility image used in access token testing. Generate constant number of requests per second, based on provided paramters.
```

Comment:
* Mention https://github.com/kubernetes/kubernetes/issues/83375, it contains a write-up on what "access token testing" means
* s/Generate/Generates/ or maybe something like "Utility can be used to generate X QPS to kube-apiserver using Y access tokens where both X, Y are configurable via flags"

Reply:
Done. Also, updated other parts of readme to support copy-paste version to test and run.